account_financial_report_qweb/README.rst

```diff
@@ -50,6 +50,7 @@ Contributors
 * Francesco Apruzzese <opencode@e-ware.org>
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
 * Julien Coux <julien.coux@camptocamp.com>
+* Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
git blame will know that, but here we put only relevant contributors, or the list would be big for all the modules.